osa/paths.py

```diff
@@ -231,7 +231,7 @@ def destination_dir(concept: str, create_dir: bool = True) -> Path:
     """
     nightdir = utils.date_to_dir(options.date)
 
-    if concept == "MUON":
+    if concept in {"MUON", "INTERLEAVED"}:
```

Comment:
if we want to place files here `/fefs/aswg/data/real/DL1/[date]/v0.10/interleaved` should be `Path(cfg.get("DL1_DIR")) / nightdir / options.prod_id / "interleaved"`

The same could be applied to muons, and datacheck files, currently placed in the same directory